staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -42,10 +42,15 @@ import (
 type AdmissionOptions struct {
 	// RecommendedPluginOrder holds an ordered list of plugin names we recommend to use by default
 	RecommendedPluginOrder []string
-	// DefaultOffPlugins a list of plugin names that should be disabled by default
-	DefaultOffPlugins []string
-	PluginNames       []string
-	ConfigFile        string
+	// DefaultEnabledPluginSet is a set of plugin names that is enabled by default
+	DefaultEnabledPluginSet sets.String
+	// DEPRECATED flag, should use EnabledAdmissionPlugins and DisabledAdmissionPlugins.
+	// They are mutually exclusive, specify both will lead to an error.
+	PluginNames []string
+
+	EnableAdmissionPlugins  []string
+	DisableAdmissionPlugins []string
+	ConfigFile              string
 
 	Plugins *admission.Plugins
```

Comment:
`Plugins` is a bad name. What is this exactly? AllPlugins?

Reply:
Yes, it is of all registered plugins.